src/routes/album.js

```diff
@@ -0,0 +1,12 @@
+// src/routes/album.js
+const express = require('express');
+const albumController = require('../controllers/album');
+const router = express.Router();
+
+router.post('/:id/albums', albumController.createAlbum);
+router.get('/', albumController.getAllAlbums);
+router.get('/:id', albumController.getAlbumById);
+router.patch('/:id', albumController.updateAlbum);
```

Comment:
To stick with the same naming, updateAlbumById, deleteAlbumById would be better.